### PR TITLE
Update certificates nugets

### PR DESCRIPTION
**Pull request by sahma19**

### CODE REVIEW

File: domains/certificates/Query.API/API.IntegrationTests/UnhealthTests.cs [hunk 1]
```diff
@@ -2,18 +2,23 @@
...
-public class UnhealthTests : TestBase, IClassFixture<QueryApiWebApplicationFactory>
+public class UnhealthTests :
+    TestBase,
+    IClassFixture<QueryApiWebApplicationFactory>,
+    IClassFixture<PostgresContainer>
 {
     private readonly QueryApiWebApplicationFactory factory;
 
-    public UnhealthTests(QueryApiWebApplicationFactory factory)
+    public UnhealthTests(QueryApiWebApplicationFactory factory, PostgresContainer dbContainer)
```
Comment (tnickelsen) [inline domains/certificates/Query.API/API.IntegrationTests/UnhealthTests.cs:17]: Why this change?

File: domains/certificates/Query.API/API/Program.cs [hunk 2]
```diff
@@ -52,7 +52,8 @@
 builder.Services.AddDbContextFactory<ApplicationDbContext>();
 
 builder.Services.AddHealthChecks()
-    .AddNpgSql(builder.Configuration.GetConnectionString("Postgres")!);
+    .AddNpgSql(sp => sp.GetRequiredService<IConfiguration>().GetConnectionString("Postgres")!);
```
Comment (tnickelsen) [inline domains/certificates/Query.API/API/Program.cs:55]: Why this change?

File: domains/certificates/RegistryConnector/Worker/Program.cs [hunk 3]
```diff
@@ -89,11 +89,11 @@
 
         cfg.ConfigureEndpoints(context);
 
-        cfg.ConfigureJsonSerializerOptions(options =>
+        cfg.ConfigureJsonSerializerOptions(jsonSerializerOptions =>
```
Comment (tnickelsen) [inline domains/certificates/RegistryConnector/Worker/Program.cs:92]: Why change the name?

File: domains/certificates/RegistryConnector/Worker/Program.cs [hunk 4]
```diff
@@ -54,7 +54,7 @@
 builder.Services.AddScoped<IKeyGenerator, KeyGenerator>();
 
 builder.Services.AddHealthChecks()
-    .AddNpgSql(builder.Configuration.GetConnectionString("Postgres")!);
+    .AddNpgSql(sp => sp.GetRequiredService<IConfiguration>().GetConnectionString("Postgres")!);
```
Comment (tnickelsen) [inline domains/certificates/RegistryConnector/Worker/Program.cs:57]: Why this change?

File: domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs [hunk 5]
```diff
@@ -2,6 +2,7 @@
 using System.ComponentModel.DataAnnotations;
 using System.Text;
 using Microsoft.Extensions.DependencyInjection;
+using ProjectOrigin.HierarchicalDeterministicKeys.Implementations;
```
Comment (tnickelsen) [inline domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs:5]: This can be deleted?

File: domains/certificates/RegistryConnector/Worker/Program.cs (hunk 4 above)
Comment (sahma19) [reply domains/certificates/RegistryConnector/Worker/Program.cs:57]: After upgrading a healthcheck nuget package, the old way does not work anymore, causing tests to fail. Therefore the serviceprovider is now used to register the npgsql health endpoint.

File: domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs (hunk 5 above)
Comment (sahma19) [reply domains/certificates/RegistryConnector/Worker/ProjectOriginOptions.cs:5]: I don't think so. I upgraded project origin, and then it wanted this dependency

File: domains/certificates/Query.API/API/Program.cs (hunk 2 above)
Comment (sahma19) [reply domains/certificates/Query.API/API/Program.cs:55]: After upgrading a healthcheck nuget package, the old way does not work anymore, causing tests to fail. Therefore the serviceprovider is now used to register the npgsql health endpoint.

File: domains/certificates/Query.API/API.IntegrationTests/UnhealthTests.cs (hunk 1 above)
Comment (sahma19) [reply domains/certificates/Query.API/API.IntegrationTests/UnhealthTests.cs:17]: This test checks if rabbitmq, returns 503, if it is not available, but due to the fact that rabbitmq uses postgresql, for some outerbox, it requires postgres to be running before it can be started, before we can check rabbitmq's health, so i added a postgresdb container to be spun up.

**Review by tnickelsen — APPROVED** [review]
